module-2/__tests__/userController.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { Request, Response, NextFunction } from 'express';
+import { StatusCodes } from 'http-status-codes';
+import request from 'supertest';
+import User from '../models/User';
+import sequelizeFixtures from 'sequelize-fixtures';
+import { users } from './fixtures/users';
+import sequelize from '../data-access';
+import { server as app } from '../app';
+import { createUserData } from './mocks';
+
+jest.mock('../middleware/checkToken', () =>
+  jest.fn((req: Request, res: Response, next: NextFunction) => next())
+);
+
+const populateDB = () => {
+  return sequelizeFixtures.loadFixtures(users, { User });
+};
+
+const clearDB = () => {
+  return User.destroy({ truncate: true, cascade: true });
+};
+
+describe('userController', () => {
+  beforeEach(() => populateDB());
+  afterEach(() => clearDB());
+  afterAll(() => {
+    sequelize.close();
+    app.close();
+  });
+
+  describe('GET /users', () => {
+    it('GET /users should return full list of users', async () => {
+      const response = await request(app).get('/users');
+
+      expect(response.status).toEqual(StatusCodes.OK);
+      expect(response.body).toHaveLength(4);
+
+      response.body.forEach((user) => {
+        expect(user).toHaveProperty('login');
+        expect(user).toHaveProperty('password');
+        expect(user).toHaveProperty('age');
+        expect(user).toHaveProperty('isDeleted');
+      });
+    });
+  });
+
+  describe('GET /users/:id', () => {
+    it('should return a specific user if exists', async () => {
+      const response = await request(app).get('/users/1');
+
+      expect(response.status).toEqual(StatusCodes.OK);
+      expect(response.body.id).toBe('1');
+      expect(response.body.login).toBe('User1');
+      expect(response.body.age).toEqual(33);
+      expect(response.body.isDeleted).toBe(false);
```

Comment:
https://jestjs.io/docs/expect#tomatchobjectobject

could be simplyfied

Reply:
Thank you for your review! I have simplified it as discussed.